core/integration/features/verified/0070-MKTD-market-decimal-places-negative.feature

```diff
@@ -0,0 +1,176 @@
+            | party0 | USD/DEC19 | buy  | 1      | 900    | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1a  |
+            | party0 | USD/DEC19 | sell | 1      | 1100   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-2a |
+            | party0 | USD/DEC22 | buy  | 1      | 900    | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1a  |
+            | party0 | USD/DEC22 | sell | 1      | 1100   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-4a |
+
+        Then the market data for the market "USD/DEC19" should be:
+            | target stake | supplied stake |
+            | 3556         | 5000           |
+        Then the market data for the market "USD/DEC20" should be:
+            | target stake | supplied stake |
+            | 3556         | 5000           |
+        Then the market data for the market "USD/DEC21" should be:
+            | target stake | supplied stake |
+            | 3556         | 5000           |
+        Then the market data for the market "USD/DEC22" should be:
+            | target stake | supplied stake |
+            | 35569        | 5000           |
+        # target stake= vol * mark price * rf = 1*1000*3.5569036*10 = 35569
+
+        When the opening auction period ends for market "USD/DEC21"
+        And the opening auction period ends for market "USD/DEC20"
+        And the opening auction period ends for market "USD/DEC19"
+        And the opening auction period ends for market "USD/DEC22"
+
+        Then the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "USD/DEC21"
+        And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "USD/DEC20"
+        And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "USD/DEC19"
+        And the trading mode should be "TRADING_MODE_OPENING_AUCTION" for the market "USD/DEC22"
+
+        Then the parties should have the following account balances:
+            | party  | asset | market id | margin | general  | bond |
+            | party0 | ETH   | USD/DEC21 | 8536   | 4881525  | 1000 |
+            | party1 | ETH   | USD/DEC21 | 1081   | 99977539 | 0    |
+            | party2 | ETH   | USD/DEC21 | 4388   | 99901468 | 0    |
+            | party0 | ETH   | USD/DEC20 | 7768   | 4881525  | 1000 |
+            | party1 | ETH   | USD/DEC20 | 1081   | 99977539 | 0    |
+            | party2 | ETH   | USD/DEC20 | 4388   | 99901468 | 0    |
+            | party0 | ETH   | USD/DEC19 | 8536   | 4881525  | 1000 |
+            | party1 | ETH   | USD/DEC19 | 1081   | 99977539 | 0    |
+            | party2 | ETH   | USD/DEC19 | 4388   | 99901468 | 0    |
+            | party0 | ETH   | USD/DEC22 | 89635  | 4881525  | 1000 |
+            | party1 | ETH   | USD/DEC22 | 19218  | 99977539 | 0    |
+            | party2 | ETH   | USD/DEC22 | 85368  | 99901468 | 0    |
+
+
+    Scenario: 002, Users engage in a USD market auction, (0070-MKTD-003, 0070-MKTD-008)
```

Comment:
> Users engage in a USD market auction 

What does this mean?

Reply:
> 0070-MKTD-008 

Ditto for this one; "Trades prices, like orders, are shown in market precision." We'd need to see what's on the event bus / data node.